### ISSUE TITLE
Tracker Language not detected correctly (Ukr)

### ISSUE THREAD

**Issue by OwlBawl** [Sonarr/Sonarr#6714]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

<img width="1265" alt="Screenshot 2024-04-12 at 10 53 40" src="https://github.com/Sonarr/Sonarr/assets/7028077/2b0fff25-7f63-425b-9f4d-c04f3752a462">

The searched item containing preferred language (Ukr) mismatched and doesn't grab it from Toloka.to tracker. 
After the download Sonarr detects correct language.

### Expected Behavior

The language itself is described in torrent name / description so the correct info source is present.

### Environment

```markdown
- OS: Unraid
- Sonarr: 4.0.3.1413
- Docker Install: Yes
- Using Reverse Proxy: Nginx
- Browser: Chrome
```

### What branch are you running?

Main

### Trace Logs?

[sonarr.trace.txt](https://github.com/Sonarr/Sonarr/files/14955977/sonarr.trace.txt)
[sonarr.debug.txt](https://github.com/Sonarr/Sonarr/files/14955980/sonarr.debug.txt)

**Comment by markus101** [comment]
`2xUkr` isn't a language, `Ukr` would be or even `2x Ukr`, but by combining it into a single string without spaces they've broken the parsing of it.

What is `2x` in this cases, multiple Ukrainian audio streams? Is this a common this this indexer does or is this a one off?

**Comment by OwlBawl** [comment]
That's not a common situation, but yes, this means Double Ukr audio streams in release.

**Comment by OwlBawl** [comment]
Yes, of course 
Search cleared up that this naming is not so frequent but was unnoticed by me previously.
<img width="861" alt="Screenshot 2024-04-12 at 17 57 45" src="https://github.com/Sonarr/Sonarr/assets/7028077/1fbb9c59-d293-4432-a826-fab9924c0812">

The maximum **N** i've found was 6x 
Wouldn't overload you with screenshots

**Comment by OwlBawl** [comment]
There is one more good private Ukr tracker who's rool by default - releases ONLY with Ukrainian audio streams. So they don't append UKR in releases name (it's by default).
So i had to apply manual rewrite rule in Prowlarr to add UKR to names. Maybe it would be correct to mark it UKR by default.
<img width="306" alt="Screenshot 2024-04-12 at 18 10 36" src="https://github.com/Sonarr/Sonarr/assets/7028077/50920356-4ed6-4e90-a2c6-a0b234722811">

**Comment by markus101** [comment]
Thanks, can you drop some of those names as text, just want to make sure we have a diverse set of tests for them.

**Comment by markus101** [comment]
Could you do the same thing, but for TV releases please.

**Comment by OwlBawl** [comment]
Sure

Аватар: Останній захисник (Сезон 1) / Avatar: The Last Airbender (Season 1) (2024) WEB-DLRip-AVC 2xUkr/Eng | Sub Ukr/Eng
Справжній детектив (Сезон 1-3) / True Detective (Season 1-3) (2014-2019) BDRip-AVC 3xUkr/Eng | Ukr/Eng
Хлопчаки / Хлопаки (Сезон 1-3) / The Boys (Seasons 1-3) (2019-2022) BDRip-AVC 4xUkr/Eng | Sub 2xUkr/Eng
Рання Пташка (Сезон 1, серії 01-26 із 51) / Erkenci Kuş (Season 1, episodes 01-26) (2018) WEBRip-AVC 2Ukr/Tur

(and also included one example with listed episodes of the season)
